pkg/cloudprovider/providers/photon/photon.go

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 // and kube-controller-manager with config flag: '--cloud-provider=photon
 // --cloud-config=[path_to_config_file]'. When running multi-node kubernetes
 // using docker, the config file should be located inside /etc/kubernetes.
+
 package photon
```

Comment:
Why add the newline?

Reply:
@liggitt the linter expect a comment on `photon` if its not separated between package name and comment. This new line nullify that expectation.

Reply:
but this comment is a package comment... right?

Reply:
@liggitt true.. If its without newline the linter says "package comment should be of the form "Package photon ...".  To avoid that expectation from linter I added a new line. please let me know if I want to discard this change.

Reply:
adding the newline also means godoc will not treat that comment as a package comment. we want this to appear in the godoc.

Reply:
Sure @liggitt , I discarded this change.